db/pf-schema-X.Y.Z.sql

```diff
@@ -1259,6 +1259,16 @@ CREATE TABLE `api_user` (
   PRIMARY KEY (`username`)
 ) ENGINE=InnoDB;
 
+--
+-- Table structure for table `keyval`
+--
+
+CREATE TABLE keyval (
```

Comment:
I think we should rename this table to something more meaningful. Are we planning to eventually use it for other key-value pairs? If not, it should have dhcp in its name.

Reply:
I think that would be a good table for distributed caching that will be usable in Golang and in Perl since CHI only works in Perl AFAIK.

But then since this is persistent, perhaps not having cache in its name either would be good.

So I'd suggest: key_value_storage

Its pretty much what fab named it, just not abbreviated and a bit more self-explanatory.